src/screens/TopicMemberScreen/index.js

```diff
@@ -177,6 +178,7 @@ const TopicMemberScreen = () => {
         setIsFirstTimeOpen={setIsFirstTimeOpen}
         setSearchText={setSearchText}
         withoutRecent={true}
+        isUser={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `isUser` is being passed to the `UsersFragment` component. Ensure that this prop is properly handled in the `UsersFragment` component and doesn't introduce any unexpected behavior.
```diff
+ isUser={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->